gapic-generator-cloud/lib/google/gapic/generators/cloud_generator.rb

```diff
@@ -54,7 +54,7 @@ def generate
           end
 
           files << cop(gen("credentials.erb",
-                           "lib/#{credentials_file_path @api}.rb",
+                           "lib/#{credentials_file_path @api}",
```

Comment:
How are delineating between .rb files and others?

Reply:
That is the difference is designated by using the suffix `_file_path` vs. `_require` (or `_require_path`).